canopen/node/local.py

```diff
@@ -127,3 +130,39 @@ def _find_object(self, index, subindex):
                 raise SdoAbortedError(0x06090011)
             obj = obj[subindex]
         return obj
+
+    def _nmt_state_changed(self, old_state, new_state):
+        if new_state == "OPERATIONAL":
+            for i, pdo in self.tpdo.map.items():
+                if pdo.enabled:
+                    try:
+                        pdo.start()
+                        logger.info(f"Successfully started TPDO {i}")
```

Comment:
This should use `pdo.name` to disambiguate between nodes.

And please stick to the formatting style used in the rest of the code.  For logger calls, we use `%`-style and pass the substitutions as arguments (where formatters see them as structured data).  For other uses, f-strings are preferred.